lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,451 @@
+---
+name: Parameters
+field_name_base_url: https://hl7.org/fhir/r4/parameters-definitions.html#Parameters
+fields:
+
+  - name: resourceType
+    required: 'Yes'
+    type: string
+    description: The type of the FHIR resource.
+    action: modify
+    example: |
+      {
+        "resourceType": "Parameters"
+      }
+    note: resourceType must be Parameters
+    url: https://hl7.org/fhir/r4/json.html#resources
+
+  - name: parameter
+    required: 'Yes'
+    type: List of BackboneElement
+    description: Parameters defined by the operation.
+    action: modify
+    url: https://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: chargeItem
+    required: 'Yes'
+    type: resource
+    description: The ChargeItem to be modified.
+    action: modify
+    example: |
+      {
+        "parameter": [
+          "name": "chargeItem",
+          "resource": {
+            "resourceType": "ChargeItem"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Resource field is required and it must be of type ChargeItem</li>
+        <li>Only one `chargeItem` parameter may be provided</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: transactionId
```

Comment:
Should we list the transactionId parameter above the chargeItem one, so the chargeItem supported fields follow the chargeItem parameter?

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/34f3cf8fd6a7e2443d2d5c8d1b297748b5b0fe6f